README.md

```diff
@@ -45,23 +51,25 @@ may also be helpful.
 
 ## Quick usage
 
+<!---
```

Comment:
TBH, it's equal to me, maybe your approach is much more efficient.

By the way, I propose to use the aforementioned `<!--- README` so that we can grep all `README.md`s to generate the TODO list: each time the given section of the README is done, the comment can be removed.

Thus, it's even better to reword it as `<!--- README [section]` with section being `LIBRARYNAME`, `Quick usage`, `Awecode` to have a precise todo